lib/FreeRTOS/freertos-plus-mqtt/freertos_mqtt_agent.c

```diff
@@ -1797,17 +1822,17 @@ MQTTStatus_t MQTTAgent_Free( MQTTContextHandle_t mqttContextHandle,
                                         pCommandCompleteCallbackContext, /* pCommandCompleteCallbackContext */
                                         NULL,                            /* incomingPublishCallback */
                                         NULL,                            /* pIncomingPublishCallbackContext */
-                                        blockTimeMS );
+                                        blockTimeMs );
     return statusReturn;
 }
 
 /*-----------------------------------------------------------*/
 
-MQTTStatus_t MQTTAgent_Terminate( uint32_t blockTimeMS )
+MQTTStatus_t MQTTAgent_Terminate( uint32_t blockTimeMs )
 {
     MQTTStatus_t statusReturn = MQTTSuccess;
 
-    statusReturn = createAndAddCommand( TERMINATE, 0, NULL, NULL, NULL, NULL, NULL, blockTimeMS );
+    statusReturn = createAndAddCommand( TERMINATE, 0, NULL, NULL, NULL, NULL, NULL, blockTimeMs );
```

Comment:
IMO this is a little unreadable. I think that technique you had of using a struct to make it clear what parameter is being assigned would come in handy here.

Reply:
Possibly, however this actually is the function that validates and assigns these parameters to the struct, and the process of "Assign to struct -> validate params -> get new struct from pool -> copy to retrieved struct" just seems wrong. "Get new struct from pool -> Assign to struct -> validate params" could work, but it still seems weird to validate after the assignment.